sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -289,20 +291,133 @@ public RecognizeEntitiesResultCollection recognizeEntitiesBatch(
+     *
+     * For a list of supported entity types, check: <a href="https://aka.ms/tanerpii">this</a>
+     * For a list of enabled languages, check: <a href="https://aka.ms/talangs">this</a>. This method will use the
+     * default language that is set using {@link TextAnalyticsClientBuilder#defaultLanguage(String)}. If none is
+     * specified, service will use 'en' as the language.
+     *
+     * <p><strong>Code Sample</strong></p>
+     * <p>Recognize the PII entities details in a document.</p>
+     *
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.recognizePiiEntities#String}
+     *
+     * @param document The document to recognize PII entities details for.
+     * For text length limits, maximum batch size, and supported text encoding, see
+     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits">data limits</a>.
+     *
+     * @return A {@link PiiEntityCollection recognized PII entities collection}.
+     *
+     * @throws NullPointerException if {@code document} is null.
+     * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PiiEntityCollection recognizePiiEntities(String document) {
+        return recognizePiiEntities(document, client.getDefaultLanguage());
+    }
+
+    /**
+     * Returns a list of Personally Identifiable Information(PII) entities in the provided document
+     * with provided language code.
+     *
+     * For a list of supported entity types, check: <a href="https://aka.ms/tanerpii">this</a>
+     * For a list of enabled languages, check: <a href="https://aka.ms/talangs">this</a>
+     *
+     * <p><strong>Code Sample</strong></p>
+     * <p>Recognizes the PII entities details in a document with a provided language code.</p>
+     *
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.recognizePiiEntities#String-String}
+     *
+     * @param document The document to recognize PII entities details for.
+     * For text length limits, maximum batch size, and supported text encoding, see
+     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits">data limits</a>.
+     * @param language The 2 letter ISO 639-1 representation of language. If not set, uses "en" for English as default.
+     *
+     * @return The {@link PiiEntityCollection recognized PII entities collection}.
+     *
+     * @throws NullPointerException if {@code document} is null.
+     * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PiiEntityCollection recognizePiiEntities(String document, String language) {
+        Objects.requireNonNull(document, "'document' cannot be null.");
+        return client.recognizePiiEntities(document, language).block();
```

Comment:
Can this call the method on L373

Reply:
same reason as in https://github.com/Azure/azure-sdk-for-java/pull/13687#discussion_r474168659